app/_hub/kong-inc/jwe-decrypt/_3.1.x.md

```diff
@@ -0,0 +1,60 @@
+---
+name: Kong JWE Decrypt
```

Comment:
Since this plugin is single sourced, it won't need this file. Anything that is specific to 3.1 can be versioned within the _index file. Ex. params that are for 3.1 and later can have `minimum_version: 3.1` and for any text that is just description text, you can use [conditional rendering](https://github.com/Kong/docs.konghq.com/blob/main/docs/single-sourced-plugins.md#conditional-rendering).

Reply:
Or since this is new for 3.1, you probably won't need any versioning/conditional rendering.

Reply:
thanks, done